### PR TITLE
Roadmap Update

### PR DESCRIPTION
**Pull request by Maria-Nosyk**

### CODE REVIEW

File: src/roadmap.md [hunk 1]
```diff
@@ -91,98 +51,129 @@ transactions and messages for Java Binding too. This step will promote all the
 corresponding [benefits](#Protobuf-as-the-Serialization-Format-in-Exonum)
 discussed above.
 
-## First Quarter of 2019
+### Java Binding: Time Service
 
-### Service Interface Standard: Read Requests
+We add another built-in Exonum service into the array of services that can be
+easily launched with an Exonum blockchain in Java – the Time Oracle. The Time
```
Comment (dmitry-timofeev) [inline src/roadmap.md]: Do we link released stuff here (../somewhere/time-oracle.md?)?

File: src/roadmap.md [hunk 2]
```diff
@@ -91,98 +51,129 @@ transactions and messages for Java Binding too. This step will promote all the
...
 
-The introduction of dynamic Java services will enable adding Java services to
-a running blockchain without the need to restart nodes. In other words, new
-services will be included into the network on the go. Support for
-dependencies between dynamic services will be added in one of the following
-versions.
+### Java Application Package
 
-### Communication between Services within the Blockchain
+A usability achievement that presupposes shipment of Exonum as a
```
Comment (dmitry-timofeev) [inline src/roadmap.md]: If it is possible, I'd simplify a little. Does presuppose work here?

File: src/roadmap.md [hunk 3]
```diff
@@ -91,98 +51,129 @@ transactions and messages for Java Binding too. This step will promote all the
...
+who plan to develop Exonum-based services. The application will have Rust and
+other related libraries pre-installed. This means that no compilation of the
+application will be required.
 
-As a method of improving the modularity of services and expanding the
-possibilities for their reuse, the Exonum team will introduce the ability for
-services to communicate with one another within one blockchain. Services will
-be able to issue and process transactions from other services and can, thus, be
-reused in a variety of scenarios.
+### Java Light Client - v.1
```
Comment (dmitry-timofeev) [inline src/roadmap.md]: v.1 might seem confusing (as the described things will be released in 0.2.0, but some more features might come in 0.2+). Maybe name it just "Java light client", and the latter section "Java Light Client Enhancements"?

File: src/roadmap.md [hunk 4]
```diff
@@ -91,98 +51,129 @@ transactions and messages for Java Binding too. This step will promote all the
...
+determine its weak spots.
 
-The functionality of private transactions implements the existence of certain
-data within the blockchain network which are known only to certain validators.
-These validators will execute private transactions locally, without changing
-the blockchain state.
+## First Quarter of 2020
 
-### Non-replayability of Transactions
+### Implementation of gRPC
```
Comment (dmitry-timofeev) [inline src/roadmap.md]: Is it confirmed? :eye:

Comment (aleksuss) [reply src/roadmap.md]: AFAIK nope

File: src/roadmap.md (hunk 1 above)
Comment (Maria-Nosyk) [reply src/roadmap.md]: I am not sure ерш is necessary as the document announces things that are yet to come at some point in time. But I can link those already released, if you want. No problem

**Pull request merged**